### ISSUE TITLE
Twitch Stream does not continue playing when user locks screen

### ISSUE THREAD

**Issue by Vegas10128** [cookpete/react-player#1505]
User views webpage with player on mobile device, starts the twitch stream in player. 

User locks phone, 
Stream stops and cannot be played without opening webpage again. 

Is there a way to continue stream when user locks screen on phone? 
Or navigate between apps?

**Comment by jaxomlotus** [comment]
In general there seems to be a bunch of incompatibilities with lock screen behavior and this app. On my end, the pause button on the lock screen is not respected by the player on iOS. I have a separate issue for that but I do wish @cookpete would comment on this.

**Comment by cookpete** [comment]
This seem to work for Twitch but only once the player has tapped on the video itself (not just tapped to unmute). 

If I tap to unmute and then lock my screen _without tapping anything else_, nothing shows. If I then unlock, tap into the player to resume the video, then lock my screen – suddenly the iOS play button shows and lets me continue listening.

For some reason it seems like iOS is not treating the initial tap to unmute as a proper "interaction" that enables the lock screen controls.

**Comment by jaxomlotus** [comment]
Just checked on Android and apparently it is an issue there too.
